tests/layer_tests/tensorflow_tests/test_tf_FloorDiv.py

```diff
@@ -49,3 +48,65 @@ def test_add_placeholder_const_1D(self, params, ie_device, precision, ir_version
                                                           use_new_frontend=use_new_frontend),
                    ie_device, precision, ir_version, temp_dir=temp_dir,
                    use_new_frontend=use_new_frontend)
+
+
+class TestFloorDivStaticInput(CommonTFLayerTest):
+    def create_flordiv_tf_net(self, min, max, step, y, dtype, ir_version, use_new_frontend, **kwargs):
+        import tensorflow as tf
+        
+        x = np.arange(min, max, step, dtype=dtype)
+        x_shape = x.reshape(kwargs['x_shape']).shape if 'x_shape' in kwargs else x.shape
```

Comment:
please extract shape from input_info dictionary (like it is done in other layers tests) instead of using kwargs.
I am afraid modifying kwarg can affect other test cases

Reply:
unfortunately until the IR is not created we don't have `input_info` dictionary id didn't modify here `kwargs`, we are only reading it and it's not modified anywhere else